pytest.ini

```diff
@@ -24,5 +24,9 @@ testpaths =
     # Llama
     forge/test/mlir/llama/test_llama_inference.py::test_llama_inference
 
+    # Benchmark
+    # MNIST Linear
+    forge/test/benchmark/benchmark/models/mnist_linear.py::test_mnist_linear
```

Comment:
Do we want to run benchmarking for every commit, or define nightly job to run it? 

Why do we need it here as it's already defined here: `.github/workflows/build-and-test.yml`?

Reply:
For now, yes, it is small model, we're a still building infrastructure. This is not real-world model, maybe I remove it at all. This is initial commit, I don't know if we can even download files, preserve results, compare them, many things I didn't put into the test, like bigger batch, data format, many things don't work in MLIR, yet, this only starting point to see if we can run CI in a way we want. We will iterate. Vlada Milosevic is also working on many things necessary to make this to work end-to-end.